JavaScriptSPA/index.html

```diff
@@ -6,24 +6,29 @@
     <title>Quickstart | MSAL.JS Vanilla JavaScript SPA</title>
     
     <!-- IE support: add promises polyfill before msal.js  -->    
-    <script type="text/javascript" src="//cdn.jsdelivr.net/npm/bluebird@3.7.2/js/browser/bluebird.min.js"></script>
-    <script type="text/javascript" src="https://alcdn.msftauth.net/lib/1.2.1/js/msal.js" integrity="sha384-9TV1245fz+BaI+VvCjMYL0YDMElLBwNS84v3mY57pXNOt6xcUYch2QLImaTahcOP" crossorigin="anonymous"></script>
+    <!-- polyfilling promise -->
+    <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/es6-promise/dist/es6-promise.auto.min.js"></script>
+    <!-- polyfilling fetch -->
+    <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/whatwg-fetch/dist/fetch.umd.min.js"></script>
+    <!-- depending on your browser you might want to include babel polyfill -->
+    <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/@babel/polyfill@7.4.4/dist/polyfill.min.js"></script>
```

Comment:
Why is this needed?

Reply:
I was adding pollyfills for ie11 support but there are a bunch of other things that still don't work and forcing a very restricted js usage (at least without something like npm and babel). 

Happily though, I've just been told by a few CxP people that aad samples can drop support for ie11. Removing this part then.